aries_vcx/src/handlers/out_of_band/receiver.rs

```diff
@@ -151,7 +152,7 @@ impl OutOfBandReceiver {
     }
 
     pub fn to_string(&self) -> VcxResult<String> {
-        self.oob.to_string()
+        self.oob.to_string().map_err(|err| err.into())
```

Comment:
```
json!(self.oob).to_string()
```
doesn't ever return error - then we could also change return value of this function to `String`. 

I've noticed we have many functions which return `VcxResult` whereas they actually never return Error. Would be nice to clean this up